osf/metadata/serializers/google_dataset_json_ld.py

```diff
@@ -0,0 +1,65 @@
+import json
+from osf.metadata import gather
+from osf.metadata.serializers import _base
+from osf.metadata.rdfutils import DCT, OSF, FOAF
+from website import settings
+
+class GoogleDatasetJsonLdSerializer(_base.MetadataSerializer):
+    mediatype = 'application/ld+json'
+
+    def filename(self, osfguid: str):
+        return f'{osfguid}-metadata.json-ld'
+
+    def serialize(self, basket: gather.Basket):
+        metadata = {
+            '@context': 'https://schema.org',
+            '@type': 'Dataset',
+            'dateCreated': next(basket[DCT.created]),
+            'dateModified': next(basket[DCT.modified]),
+            'name': next(basket[DCT.title]),
+            'description': next(basket[DCT.description], None),
+            'url': next(basket[DCT.identifier]),
+            'keywords': [keyword for keyword in basket[OSF.keyword]],
+            'publisher': {
+                '@type': 'Organization',
+                'name': 'Center For Open Science'
+            },
+            'creator': format_creators(basket),
+            'identifier': [keyword for keyword in basket[DCT.identifier]],
+            'license': next(basket[DCT.rights], None)
+        }
+
+        if basket.osf_type == 'Registration':
```

Comment:
What about on line `osf/metadata/serializers/datacite_json.py` on line 327? I was looking at that to determine if this was suitable. [line](https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/metadata/serializers/datacite_json.py#L327-L328) I can see this is a different situation, but it shouln't be ok to check againest a litteral string in either case, IMO. What is your reasoning? Do you see this as inconsistent?

Reply:
yeah i agree, same situation -- would be better if that line 327 checked  `focustype == OSF.Registration` instead of the de-namespaced name and a magic string, good catch